### ISSUE TITLE
SA1114: False positive on argument list with directives

### ISSUE THREAD

**Issue by nxtn** [DotNetAnalyzers/StyleCopAnalyzers#3284]
To be fixed by #3283 

```cs
    TestMethod(
#if TESTSYMBOL
        arg1);
#else
        arg2);
#endif
```
